docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -12,6 +12,32 @@ If your problem isn’t described here, please review open issues in the followi
 
 Have a question? Contact us in the https://discuss.elastic.co/[discuss forum].
 
+[float]
+[[rule-cannot-decrypt-api-key]]
+=== Rule cannot decrypt apiKey
+
+*Problem*:
+
+The rule fails to execute and has an `Unable to decrypt attribute "apiKey"` error.
+
+*Solution*:
+
+This error happens when the `xpack.encryptedSavedObjects.encryptionKey` value used to create the rule does not match the value used during rule execution. Depending on the scenario, there are different ways to solve this problem:
+
+[cols="2*<"]
+|===
+
+| If the value in `xpack.encryptedSavedObjects.encryptionKey` was manually changed, and the previous encryption key is still known.
+| Ensure any previous encryption key is part of the <<encryption-key-rotation, decryption only key>>.
+
+| If another {kib} instance with a different encryption key connects to the cluster.
+| It might be trying to run the rule using a different encryption key than what the rule was created with. Ensure the encryption keys between all the {kib} instances are the same is necessary, and setting <<encryption-key-rotation, decryption only keys>> for previously used encryption keys.
+
+| If other scenarios don't apply.
+| Generate a new API key for the rule by disabling/enabling the rule.
```

Comment:
```suggestion
| Generate a new API key for the rule by disabling or enabling the rule.
```

Reply:
Hmm, the steps I'm trying to describe here is to disable then enable the rule. Maybe `Generate a new API key for the rule by disabling then enabling the rule`?